src/components/cylc/Drawer.vue

```diff
@@ -82,22 +82,26 @@ import Header from '@/components/cylc/Header'
 import { mapState } from 'vuex'
 import GScan from '@/components/cylc/gscan/GScan'
 import { mdiHome, mdiGraphql } from '@mdi/js'
+import { version } from '@/../package.json'
```

Comment:
Easier to just import it directly from `package.json` and use it in the component's `data` section, rather than needing to use Vuex (more expensive than 1 import).